README.md

```diff
@@ -138,11 +138,11 @@ Captcha::create();
 
 # Return URL
 ```php
-captcha_src();
+captcha_src('default');
 ```
 or
 ```
-Captcha::src();
```

Comment:
Thank you for the feedback and merge request @mwllgr , but this way is not suitable for use consistency. I've updated to continue using it as it is.

https://github.com/mewebstudio/captcha/commit/adb61c06e7a8e258341621522140a927ee33c116

Can you fix it and send request it back?

Reply:
Fixed it.